### PR TITLE
Fuerte simplify

### PR DESCRIPTION
**Pull request by graetzer**
Fuerte connections no longer perform reconnects internally which should make it easier to reason about it.

http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr/11191/

### CODE REVIEW

File: arangod/GeneralServer/VstCommTask.cpp [hunk 1]
```diff
@@ -400,6 +400,9 @@ void VstCommTask<T>::doWrite() {
   std::unique_ptr<ResponseItem> item(tmp);
 
   DTraceVstCommTaskBeforeAsyncWrite((size_t) this);
+  
+  // TODO: should we have a better timeout mechanism?
+//  this->setTimeout(GeneralCommTask<T>::DefaultTimeout);
```
Comment (danielhlarkin) [inline arangod/GeneralServer/VstCommTask.cpp:405]:
```suggestion
```

File: arangod/Network/NetworkFeature.cpp [hunk 2]
```diff
@@ -134,14 +134,13 @@ void NetworkFeature::prepare() {
   config.clusterInfo = ci;
   config.name = "ClusterComm";
 
-  if (_protocol == "http") {
+  config.protocol = fuerte::ProtocolType::Http;
+  if (_protocol == "http" || _protocol == "h1") {
     config.protocol = fuerte::ProtocolType::Http;
   } else if (_protocol == "http2" || _protocol == "h2") {
```
Comment (danielhlarkin) [inline arangod/Network/NetworkFeature.cpp:140]:
```suggestion
  if (_protocol == "http2" || _protocol == "h2") {
```

**Comment by graetzer** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr/11232/

**Review by maierlars — APPROVED** [review]
Models vs Code of *Connection LGTM

**Comment by jsteemann** [comment]
http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/11611/

**Pull request merged**